### PR TITLE
Implementing of FAT-4985

### PR DESCRIPTION
**Pull request by IhorBohdan**
***
[FAT-4985](https://issues.folio.org/browse/FAT-4985)
***
[C375077](https://foliotest.testrail.io/index.php?/cases/view/375077)
***
[Allure](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1614/allure/)
***

### CODE REVIEW

File: cypress/e2e/inventory/search/browse-search-permissions.cy.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    let secondUser;
+
+  before(() => {
+    cy.createTempUser([
+        permissions.uiInventoryViewInstances.gui,
+      ]).then(userProperties => {
+        firstUser = userProperties;
+      });
+    cy.createTempUser([
+        permissions.uiInventoryViewInstances.gui,
```
Comment (MaksimTerekhov) [inline cypress/e2e/inventory/search/browse-search-permissions.cy.js:19]: I see in test case
"A user with "Inventory: All permissions" permission is logged in"

So I assume here should be inventoryAll isntead of uiInventoryViewInstances

File: cypress/e2e/inventory/search/browse-search-permissions.cy.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import TestTypes from '../../../support/dictionary/testTypes';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import devTeams from '../../../support/dictionary/devTeams';
+import Users from '../../../support/fragments/users/users';
+import permissions from '../../../support/dictionary/permissions';
+
+describe('permissions: inventory', () => {
+    let firstUser;
+    let secondUser;
```
Comment (MaksimTerekhov) [inline cypress/e2e/inventory/search/browse-search-permissions.cy.js:10]: I suggest to rename user to userWithAllPermissions and userWithOnlyViewPermissions

Comment (IhorBohdan) [reply cypress/e2e/inventory/search/browse-search-permissions.cy.js]: Done

File: cypress/e2e/inventory/search/browse-search-permissions.cy.js (hunk 1 above)
Comment (IhorBohdan) [reply cypress/e2e/inventory/search/browse-search-permissions.cy.js]: Done

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/inventory/search/browse-search-permissions.cy.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  it('C375077 User with "Inventory: All permissions" permission can see browse call numbers and subjects without assigning specific browse permissions (Orchid+) (thunderjet)', { tags: [TestTypes.smoke, devTeams.thunderjet] }, () => {
+    cy.login(userWithAllPermissions.username, userWithAllPermissions.password);
+    cy.visit(TopMenu.inventoryPath);
+    InventorySearchAndFilter.switchToBrowseTab();
+    InventorySearchAndFilter.selectBrowseCallNumbers();
+    InventorySearchAndFilter.browseSearch('K1');
+    InventorySearchAndFilter.verifyCallNumberBrowsePane();
+    InventorySearchAndFilter.selectBrowseSubjects();
+    InventorySearchAndFilter.browseSearch('art');
+    InventorySearchAndFilter.verifyCallNumberBrowsePane();
```
Comment (penfriend) [inline cypress/e2e/inventory/search/browse-search-permissions.cy.js:51]: you have not verified that the search actually works and the entered result is displayed

Comment (IhorBohdan) [reply cypress/e2e/inventory/search/browse-search-permissions.cy.js:51]: When you search for this parameter a sheet with a search list is displayed (but under this parameter it finds a lot of different entities in this list). I check that the list is displayed after the search.

Comment (IhorBohdan) [reply cypress/e2e/inventory/search/browse-search-permissions.cy.js]: Fix it for verifySubjectsResultsInBrowsePane() and verifyCallNumbersResultsInBrowsePane() methods

**Review by TetianaParanich — APPROVED** [review]

File: cypress/e2e/inventory/search/browse-search-permissions.cy.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ describe('permissions: inventory', () => {
```
Comment (penfriend) [inline cypress/e2e/inventory/search/browse-search-permissions.cy.js]: please fix here linter issues

File: cypress/e2e/inventory/search/browse-search-permissions.cy.js (hunk 3 above)
Comment (penfriend) [reply cypress/e2e/inventory/search/browse-search-permissions.cy.js]: Please use these methods in autotest cypress\e2e\inventory\search\browse-search-permissions.cy.js

File: cypress/e2e/inventory/search/browse-search-permissions.cy.js (hunk 4 above)
Comment (IhorBohdan) [reply cypress/e2e/inventory/search/browse-search-permissions.cy.js:11]: Done

File: cypress/e2e/inventory/search/browse-search-permissions.cy.js (hunk 3 above)
Comment (IhorBohdan) [reply cypress/e2e/inventory/search/browse-search-permissions.cy.js]: Done

**Review by penfriend — APPROVED** [review]

**Pull request merged**